src/python/aqueduct_executor/operators/utils/utils.py

```diff
@@ -261,6 +261,10 @@ def delete_object(name: str, delete_fn: Callable[[str], None]) -> SavedObjectDel
 def write_delete_saved_objects_results(
     storage: Storage, path: str, results: Dict[str, List[SavedObjectDelete]]
 ) -> None:
+    # `[object].json()`` gives me the json as a string which gets escaped in `json.dumps()``.
```

Comment:
should we use `"""` strings to be consistent with other changes?

Reply:
I used `"""` because I was explaining the whole function. Since this is explaining a line in the code, I just used `#` instead.
Looking at [this Python comment guide](https://www.askpython.com/python/python-comments), we can also comment with `'''` but we haven't done that in this codebase.